### PR TITLE
Prac 08 feedback

### PR DESCRIPTION
**Pull request by CameronOBrien95**
Could you please check and review my code @CaitlinBevan :)

### CODE REVIEW

File: prac_08/silver_service_taxi.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+from prac_08.taxi import Taxi
+
```
Comment (CaitlinBevan) [inline prac_08/silver_service_taxi.py:2]: You should have two enters after an import

File: prac_08/silver_service_taxi.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+class SilverServiceTaxi(Taxi):
+
```
Comment (CaitlinBevan) [inline prac_08/silver_service_taxi.py:3]: Should have a docstring for all Classes, methods, functions

File: prac_08/unreliable_car.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+from prac_08.car import Car
+from random import randint
+
```
Comment (CaitlinBevan) [inline prac_08/unreliable_car.py:3]: You only need two enters after the imports

**Review by CaitlinBevan — COMMENTED** [review]
Hi @CameronOBrien95.  I think your code is looking good.  You're just missing some docstrings and some spacing is incorrect.  In Python you can use ctrl alt l for formatting (on windows) and the formatting error should have a squiggly underline.  Hope that helps :)

File: prac_08/taxi_simulator.py [hunk 4]
```diff
@@ -3,18 +3,36 @@
...
+
+
 def main():
     taxis = [Taxi("Prius", 100), Taxi("Kia", 150), SilverServiceTaxi("Hummer", 200, 4)
         ,SilverServiceTaxi("Limo", 250, 6)]
     total_bill = 0
-    print(MENU)
-    choice = int(input("your choice"))
+    choice = int(input("{}\n>>>".format(MENU)))
+    current_taxi = "none"
```
Comment (CaitlinBevan) [inline prac_08/taxi_simulator.py:13]: `current_taxi =None`

Comment (CaitlinBevan) [inline prac_08/taxi_simulator.py]: In the question it mentions current_taxi = None
```suggestion
    current_taxi = None
```

File: prac_08/taxi_simulator.py [hunk 5]
```diff
@@ -3,18 +3,36 @@
 from prac_08.taxi import Taxi
 
 MENU = "1) choose taxi\n2) Drive\n3) Bill\n4) Quit"
```
Comment (CaitlinBevan) [inline prac_08/taxi_simulator.py]: I would use what was given to us in the sample output
```suggestion
MENU = "q)uit, c)hoose taxi, d)rive"
```

File: prac_08/taxi_simulator.py [hunk 6]
```diff
@@ -3,18 +3,36 @@
...
+
 def main():
     taxis = [Taxi("Prius", 100), Taxi("Kia", 150), SilverServiceTaxi("Hummer", 200, 4)
         ,SilverServiceTaxi("Limo", 250, 6)]
     total_bill = 0
-    print(MENU)
-    choice = int(input("your choice"))
+    choice = int(input("{}\n>>>".format(MENU)))
+    current_taxi = "none"
     while choice != 4:
```
Comment (CaitlinBevan) [inline prac_08/taxi_simulator.py]: Again I would use what was  in the sample output :)
```suggestion
    while choice != "q":
```

**Comment by CameronOBrien95** [comment]
Thank you, I have made the changes and merged the files.

**Pull request merged**